### ISSUE TITLE
CreateDateColumn and UpdateDateColumn don't work with mongodb

### ISSUE THREAD

**Issue by goodoldneon** [typeorm/typeorm#6946]
**Issue type:**

[X] bug report
[ ] feature request
[ ] documentation issue

**Database system/driver:**

[ ] `cordova`
[X] `mongodb`
[ ] `mssql`
[ ] `mysql` / `mariadb`
[ ] `oracle`
[ ] `postgres`
[ ] `cockroachdb`
[ ] `sqlite`
[ ] `sqljs`
[ ] `react-native`
[ ] `expo`

**TypeORM version:**

[ ] `latest`
[ ] `@next`
[X] `0.2.28` (or put your version here)

**Steps to reproduce or a small repository showing the problem:**

Entity:

```ts
import {
  Column,
  CreateDateColumn,
  Entity,
  ObjectIdColumn,
  UpdateDateColumn,
} from 'typeorm';

@Entity()
export class Plant {
  @ObjectIdColumn()
  id: string;

  @CreateDateColumn()
  createDt: Date;

  @UpdateDateColumn()
  updateDt: Date;

  @Column()
  genus: string;

  @Column()
  species: string;
}
```

The records inserted into the DB are missing the `createDt` and `updateDt` fields.

Side note: TypeORM is seriously amazing. Keep up the great work!

<!--
    To answer those questions you need to put "x" inside the square brackets, for example:
    [x] `mysql`
    [ ] `postgres`

    Also, please format your code properly (by taking code blocks into ```ts .... ```)
--!>

**Comment by imnotjames** [comment]
Duplicate of #6788

**Comment by imnotjames** [comment]
TLDR these were never implemented for MongoDB.  They could be, but nobody has done it.
